app/src/main/java/org/fossasia/openevent/general/order/OrderDetailsViewHolder.kt

```diff
@@ -22,5 +26,12 @@ class OrderDetailsViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView)
         itemView.orderIdentifier.text = orderIdentifier
         itemView.eventSummary.text = event?.description
         itemView.organizer.text = event?.organizerName
+
+        try {
+            val bitMatrix = MultiFormatWriter().encode(attendee.email, BarcodeFormat.QR_CODE, 200, 200)
```

Comment:
@iamareebjamal of what value QR should be generated so that it could be checked in using orga app?